cmd/validate_image.go

```diff
@@ -225,12 +219,12 @@ Use an inline EnterpriseContractPolicy spec
 		"path to ApplicationSnapshot Spec JSON file")
 	cmd.Flags().StringVarP(&data.input, "json-input", "j", data.input,
 		"JSON represenation of an ApplicationSnapshot Spec")
-	cmd.Flags().StringVarP(&data.output, "output-file", "o", data.output,
-		"write output to a file. Use empty string for stdout, default behavior")
+	cmd.Flags().StringSliceVar(&data.output, "output", data.output,
+		"write output to a file in a specific format. Use empty string path for stdout. May be used multiple times. Possible formats are json, yaml, hacbs, and summary")
```

Comment:
Is summary just the one word "failure" or "success"? If so then maybe that could be made more clear somehow.

Reply:
It's the "condensed" version of the output. I think we should probably remove it, but didn't want to make that decision here.